### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: mass_editing/__openerp__.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
     ],
     "category": "Tools",
     "website": "http://www.serpentcs.com",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline mass_editing/__openerp__.py]: GPL-3 or later

File: server_environment/__openerp__.py [hunk 2]
```diff
@@ -77,6 +77,7 @@
     serv_config.get('external_service.ftp', 'tls')
     """,
     "website": "http://www.camptocamp.com",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline server_environment/__openerp__.py]: GPL-3 or later

File: server_environment_files_sample/__openerp__.py [hunk 3]
```diff
@@ -41,6 +41,7 @@
 variable of the OpenERP configuration file.
     """,
     "website": "http://www.camptocamp.com",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline server_environment_files_sample/__openerp__.py]: GPL-3 or later

**Comment by dreispt** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1:

**Pull request merged**